spec/griddler/cloudmailin/adapter_spec.rb

```diff
@@ -20,6 +41,17 @@
     })
   end
 
+  it 'normalizes parameters where the recipient has been BCCed' do
+    expect(Griddler::Cloudmailin::Adapter.normalize_params(bcc_params)).to be_normalized_to({
+      to: ['Some Identifier <some-identifier@example.com>'],
+      cc: ['emily@example.com'],
+      bcc: ['sandra@example.com'],
+      from: 'Joe User <joeuser@example.com>',
+      subject: 'Re: [ThisApp] That thing',
+      text: /Dear bob/
+    })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.